ffi/ffi_wrapper.go

```diff
@@ -362,10 +382,12 @@ func createIdentity(
 
 	resolver := register.NewDefaultRestResolverClient(addr)
 	var id register.RegisteredIdentity
-	if isUser {
+	if idDoc == idDocUser {
```

Comment:
Since have all oe fhtme defined - use switch + default+error when unknown type instead of `else` to imply third thing.

Reply:
done